advanced/kv_umbrella/apps/kv/lib/registry.ex

```diff
@@ -30,11 +30,11 @@ defmodule KV.Registry do
     end
   end
 
-  @doc """
-  Looks up for all the buckets
+  # @doc """
+  # Looks up for all the buckets
 
-  Returns %{name => PID, ...}
-  """
+  # Returns %{name => PID, ...}
+  # """
```

Comment:
Why did you comment out this?

Reply:
Because i commented a function right below, and then the @doc shows me a warning cause there is no function, but i forgot delete that sorry, ill delete it right now.